### ISSUE TITLE
Cutter produces incorrect very low resolution result from 2d mercator map

### ISSUE THREAD

**Issue by tst-blaird** [google/earthenterprise#1012]
I created a 2d mercator map of San Francisco using the tutorial data.  I used cutter and manually created the polygon by outlining the high resolution area of north east SF.  I specified a region level of 18 and left the other settings at their default values.  When I register this portable map in the GEE server and then view it everything looks correct.  See image 1-ResultOfSfCut.png.

I then created a 2d mercator map of the Orlando theme park area using imagery from the USGS Earth Explorer site.  I used cutter and manually created the polygon by outlining a rectangular area that included Disney's Magic Kingdom and Epcot.  I specified a region level of 18 and left the other settings at their default values.  See image 2-CreatingOrlandoCut.png.  When I register this portable map in the GEE server and then view it the polygon contains very low resolution imagery.  See image 3-ResultIsLowRes.png.

![1-resultofsfcut](https://user-images.githubusercontent.com/40216280/46027550-20b64180-c0bc-11e8-94b8-a1b6cd34da01.png)
![2-creatingorlandocut](https://user-images.githubusercontent.com/40216280/46027551-20b64180-c0bc-11e8-9064-0534cb9e13d5.png)
![3-resultislowres](https://user-images.githubusercontent.com/40216280/46027552-20b64180-c0bc-11e8-92dd-2702e7105394.png)

One difference that might be worth noting is the Orlando imagery contains a mosaic.  If you look at image 2-CreatingOrlandoCut.png, the lighter green area outside of the polygon is from a single imagery resource created as a mosaic from nine images.

**Comment by tst-ccamp** [comment]
@tst-blaird Just a guess, can you check your "Advanced Settings" and make sure the polygon level goes up to 18 as well? Also, try specify a zoom level for world also, like 8 or 10.

**Comment by tst-blaird** [comment]
@tst-ccamp I checked the advanced settings and the polygon level defaulted to 18.  I generated a new cutfile after expanding the advanced settings section and the result was the same.

I then specified a world level of 8 and generated a new cutfile.  The polygon was still low resolution but it appeared the tiles outside the polygon region were one level better resolution.  There were four squares in the world region for every one square in the polygon region.  Screenshot:

![4-resultislowres-worldlevel8](https://user-images.githubusercontent.com/40216280/46036634-fd978c00-c0d3-11e8-8ddd-8f6d10ad8041.png)
